README.md

```diff
@@ -1,53 +1,34 @@
-
-# Requirements
+The Shopify theme gem is a command line tool that allows you to manage and edit theme files directly on your computer. This allows you to use your preferred code editing software opposed to the built in code editor available in the “Edit HTML/CSS” page of your store. Files can be manually or automatically uploaded, downloaded, and synced. Depending on how you develop themes and your personal preference, this can allow for a more enjoyable and streamlined workflow.  If the command line is scary for you, check out the [Desktop Theme Editor app](http://apps.shopify.com/desktop-theme-editor).
 
-This gem works with OS X or Windows with Ruby 1.9. 
 
-First time installing Ruby on windows? Try [Rubyinstaller](http://http://rubyinstaller.org/). 
+## Requirements
+This gem works with OS X or Windows with Ruby 1.9.
+First time installing Ruby on windows? Try Rubyinstaller.
 
-# Installation
 
-To install the shopify_theme gem use 'gem install' (you might have use 'sudo gem install')
-
-```
-gem install shopify_theme [optional_theme_id]
-```
+## Configuration
+### Setting up Shopify Theme
 
-to update to the latest version
+First, you will need to set up a new private app to generate an API key and password. Go to [your-store-url].com/admin/apps/private in your web browser.
 
-```
-gem update shopify_theme
-```
+Click on “Create a Private App” to generate the credentials for a new app. You will need the API key and the password of this newly generated app:
 
-# Usage
+![api-key-and-password](https://cloud.githubusercontent.com/assets/5488872/5901847/958068e4-a543-11e4-8eb4-8f139cc4384c.jpg)
 
-Generate the config file. Go get a valid api_key and password for your store head to `https://[your store].myshopify.com/admin/apps/private` and generate a private application. By default it adds the main theme, if you want to edit one of your other themes, add the `theme_id`.
+In the command line, go to the directory of your theme files, or the directly where you would like your theme files to be.
 
-```
-theme configure api_key password store_url
-```
+In the command line run: 
+`theme configure api_key password store` 
+This populates the config.yml file with your app credentials. You can also manually edit the config.yml file and paste the API key and password in the appropriate area.
 
-Example of config.yml. Notice store has no http or https declaration. You can
-use `:whitelist_files:` to specify files for upload. The `assets/`, `config/`,
-`layout/`, `snippets/`, `templates/` and `locales/`directories are included by
-default.
 
-You can also use `:ignore_files:` to exclude files from getting uploaded, for
-example your `config/settings.html` or other configuration driven items
+### The config.yml File
+The config.yml file contains the information needed for Shopify to authenticate requests and edit/update the theme files in the manner specified. Take a look at this sample config.yml file for reference:
```

Comment:
Here is an example of what the contents in a typical `config.yml` file would look like: